### DISCUSSION TITLE
Support for sending the interactive messages on WhatsApp

### DISCUSSION THREAD

**Discussion by pankaj-ag** [nyaruka/floweditor#1160] [Ideas]
WhatsApp recently released a new feature called interactive messages. Now we can send the list and buttons explicitly using WhatsApp business APIs. 

https://developers.facebook.com/docs/whatsapp/guides/interactive-messages

Is there any plan for Nyaruka to add the interactive message feature in this library?

**Comment by dlobo** [comment]
We've added support for this in our fork of flow editor (we actually treat it similarly to WhatsApp message templates) and added a UI within Glific to allow folks to create list messages and/or quick replies there, that we expose to floweditor. Happy to share the code (its already on GitHub) if interested
